airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -146,6 +147,7 @@ def is_current_process_wrapped_by_dagger_run() -> bool:
 @click.option("--enable-update-check/--disable-update-check", default=True)
 @click.option("--enable-auto-update/--disable-auto-update", default=True)
```

Comment:
Out of curiosity, please tell me that Airbyte CI skips update check when it's running in CI. It skips it, right?

Reply:
[It skips it.](https://github.com/airbytehq/airbyte/blob/0f3e63bd45cf9209a31b96f048503143c3421b9a/.github/actions/run-airbyte-ci/action.yml#L118)